src/test/java/org/jpeek/skeleton/XmlClassTest.java

```diff
@@ -65,6 +66,18 @@ public void parsesMethodVisibility() {
         );
     }
 
+    @Test
+    public void thereIsNoAttributePublic() {
+        MatcherAssert.assertThat(
+            XhtmlMatchers.xhtml(
+                this.classAsXml("ClassWithDifferentMethodVisibilities")
+            ),
+            XhtmlMatchers.hasXPaths(
+                "/class/methods/method[not (@public)]"
```

Comment:
@GnusinPavel and the tests for the others visibility attributes?

Reply:
there is one already, https://github.com/yegor256/jpeek/blob/d0e9c60d253466d1e41fc79a7735777a42385745/src/test/java/org/jpeek/skeleton/XmlClassTest.java#L54-L67